plotly/offline/offline.py

```diff
@@ -298,3 +309,128 @@ def plot(figure_or_data,
             ])
         else:
             return plot_html
+
+
+def plot_mpl(mpl_fig, resize=False, strip_style=False,
+             verbose=False, **kwargs):
+    '''
+    Convert a matplotlib figure to a plotly graph locally as an HTML document
+    or string
+
+    For more information on converting matplotlib visualizations to plotly
+    graphs, call help(plotly.tools.mpl_to_plotly)
+
+    For more information on creating plotly charts locally as an HTML document
+    or string, call help(plotly.offline.plot)
+
+    :param (matplotlib figure) mpl_fig: matplotlib figure to convert to a
+        plotly graph
+    :param (bool) resize: Default = False
+    :param (bool) strip_style: Default = False
+    :param (bool) verbose: Default = False
+    :param kwargs: kwargs passed through `plotly.offline.plot`.
+        For more information on valid kwargs call `help(plotly.offline.plot)`
+    :return (None|string): if `output_type` is 'file' (default), then the graph
```

Comment:
:dolls: I think this always returns a `str`, right? It should be `:return (str):`?

https://github.com/plotly/plotly.py/blob/master/plotly/offline/offline.py#L262-L300